### ISSUE TITLE
Warning: Using <Image> with children is deprecated.

### ISSUE THREAD

**Issue by travismgibson** [itsnubix/react-native-video-controls#55]
I am seeing the following warning:

**Using <Image> with children is deprecated and will be an error in the near future. Please reconsider the layout or use <ImageBackground> instead.**

Using: 
	"react": "16.0.0-beta.5",
        "react-native": "0.49.3",
	"react-native-video-controls": "^1.4.1"

https://github.com/itsnubix/react-native-video-controls/blob/ae31712abb3dbb67a2ce3f6c17ff4d8233cc8135/VideoPlayer.js#L817

https://github.com/itsnubix/react-native-video-controls/blob/ae31712abb3dbb67a2ce3f6c17ff4d8233cc8135/VideoPlayer.js#L906

**Comment by kylemilloy** [comment]
Resolved in 1.5.1...thanks for bringing it up.

**Comment by travismgibson** [comment]
Thanks for the quick follow up.

**Comment by rossmartin** [comment]
@kylemilloy 

Thanks for being quick on this issue.  When you added `ImageBackground` via 5f41a370a2d0ab3e74e13d306b2aff4a7cdafecc it breaks this component for anyone using RN 0.45 and lower.  My app is using 0.45 and I had to install this package @1.5.0 in order for it to work.  I kept getting the dreaded "you likely forgot to export your component from the file it's defined in".  Just a heads up - I think if you mention to use 1.5.0 and lower if you're using RN 0.45 or lower is a reasonable solution.

Thanks for this component.
